digits/model/tasks/caffe_train.py

```diff
@@ -39,6 +39,50 @@
 CAFFE_DEPLOY_FILE = 'deploy.prototxt'
 
 @subclass
+class DigitsTransformer(caffe.io.Transformer):
```

Comment:
I'm nervous about what could happen if Caffe makes changes to this class.
https://github.com/BVLC/caffe/commits/master/python/caffe/io.py